spec/Section 3 -- Type System.md

```diff
@@ -1905,18 +1905,17 @@ Following are examples of result coercion with various types and values:
 The GraphQL Semantic-Non-Null type is an alternative to the GraphQL Non-Null
 type to disallow null unless accompanied by a field error. This type wraps an
 underlying type, and this type acts identically to that wrapped type, with the
-exception that {null} will result in a field error being raised. A leading
-exclamation mark is used to denote a field that uses a Semantic-Non-Null type
-like this: `name: !String`.
-
+exception that {null} will result in a field error being raised. Semantic-Non-Null
+types are only valid in a document that also contains a `@SemanticNullability`
+document-level directive. In such a document, Semantic-Non-Null fields have no 
+adornment, much like nullable fields in documents without that directive. 
```

Comment:
The type system should be valid without reference to documents; support for SDL is optional.